wheel/chia_rs.pyi

```diff
@@ -31,7 +31,7 @@ class LazyNode:
     def atom() -> bytes: ...
 
 def serialized_length(program: bytes) -> int: ...
-def tree_hash(program: bytes) -> bytes32: ...
+def tree_hash(program: ReadableBuffer) -> bytes32: ...
```

Comment:
I wasn't able to find `ReadableBuffer` declared or imported. Does this work with mypy?

Reply:
it's on line 9 in this file. It's a poor-man's definition of a python buffer.

Reply:
the reason this changed here was that I forgot to regenerate this file when merging this change

Reply:
Ah, got it, sorry, not sure how I missed it.